mapswipe_workers/mapswipe_workers/auth.py

```diff
@@ -47,17 +47,20 @@ def firebaseDB() -> object:
 class postgresDB(object):
     """Helper class for Postgres interactions"""
 
-    _db_connection = None
+    __db_connection = None
```

Comment:
it's not really clear to me why this has changed here.

Reply:
This was done for fixing pyright issues. It's an optional change.
![image](https://user-images.githubusercontent.com/7059255/195042396-3e5ed842-64e7-4366-a031-fb452ec5b18e.png)

Reply:
Also, the reason was for defining _db_connectin as a property (for using _db_connectin as an object field instead of a class field). This is used in a bit hacky way to resolve DB connection open/close issues. Maybe we can discuss this further for a better test framework.